server/src/account.rs

```diff
@@ -142,6 +111,7 @@ impl Device {
         auth_token: Vec<u8>,
         salt: String,
         registration_id: u32,
+        pni_registration_id: u32,
```

Comment:
These should be u8 to match the SignalMessage type, otherwise, we might panic due to a demote